semgrep_output_v1.atd

```diff
@@ -685,6 +685,7 @@ type matching_operation <ocaml attr="deriving show { with_path = false}"> = [
   | And
   | Or
   | Inside
+  | Anywhere
```

Comment:
we usually try to indicate when something was introduced, like (* since semgrep 1.49 *)

Reply:
I'll put up another PR to add a note.